<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arboles Equipo:DEMM</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditValue">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInsert">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insertar nodo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInOrder">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recorrido en orden</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetTraversal">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>61</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPreOrder">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>420</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recorrido preorden</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPostOrder">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>460</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Recorrido postorden</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Arboles , creacion y recorridos en orden, preorden y posorden</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSearch">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>470</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar Valor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearchResult">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
